exporter/opentelemetry-exporter-datadog/setup.py

```diff
@@ -11,13 +11,14 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
 import os
 
 import setuptools
 
 BASE_DIR = os.path.dirname(__file__)
 VERSION_FILENAME = os.path.join(
-    BASE_DIR, "src", "opentelemetry", "ext", "otlp", "version.py"
+    BASE_DIR, "src", "opentelemetry", "exporter", "datadog", "version.py"
```

Comment:
any idea what happened here? looks like theres a few instances of files being moved to different exporters in the PR

Reply:
I'm not sure what happened here. Maybe github has a weird way of tracking diffs? Anyways, the resulting files look correct and all the tests pass so the directories must be pointing to the right things.